R/render_release.R

```diff
@@ -14,9 +14,9 @@
 #' @param output_root A character string giving the root folder without version
 #' number. Default is "publish".
 #'
-#' @details The links to earlier releases and older versions can be found
-#' through a subfolder structure below `publish/version_number` that mirrors the
-#' structure of `source/`.
+#' @details The website also contains a welcoming page, a news page and separate
```

Comment:
Why this `also`, where does it refer to?  As this sentence is not connected to previous sentences, it would be better to give some more context.  E.g. Additionally to xxx, the website contains a welcoming page,...

Reply:
fixed in 45d47b5